src/public/shared.js

```diff
@@ -210,3 +129,48 @@ Area.prototype = {
 }
 
 /* -------- End Area Class -------- */
+
+
+
+/* -------- Config Class -------- */
+
+/**
+ * Config Class
+ * @param {Config} Config
```

Comment:
Config is actually no used in client.js.
Maybe in server.js ? Or will you use it in client after ?